lib/matplotlib/axes/_axes.py

```diff
@@ -5875,10 +5875,10 @@ def table(self, **kwargs):
     #### Data analysis
 
     @_preprocess_data(replace_names=["x", 'weights'], label_namer="x")
-    def hist(self, x, bins=None, range=None, normed=False, weights=None,
+    def hist(self, x, bins=None, range=None, density=None, weights=None,
```

Comment:
Changing the order is _probably_ not a good idea, for backwards compatibility's sake, even though people really shouldn't be expecting order-only arguments to work this far in.